python/servo/devenv_commands.py

```diff
@@ -11,6 +11,10 @@
 from os import path, getcwd, listdir
 
 import sys
+import toml
+import json
+import httplib
+import colorama
```

Comment:
Any import at the top level slows down all `./mach` invocations; please import them inside the method.